beta-src/src/webDiplomacyTheme.ts

```diff
@@ -1,13 +1,40 @@
 import { createTheme } from "@mui/material/styles";
 
+declare module "@mui/material/styles" {
+  interface BreakpointOverrides {
+    xs: false;
+    sm: false;
+    md: false;
+    lg: false;
+    xl: false;
+    mobile: true;
+    mobileLg: true;
+    tablet: true;
+    desktop: true;
+  }
+
+  interface TypographyVariants {
+    label: React.CSSProperties;
+  }
+
+  export interface TypographyVariantsOptions {
+    label?: React.CSSProperties;
+  }
+}
+
+declare module "@mui/material/Typography" {
+  interface TypographyPropsVariantOverrides {
+    label: true;
+  }
+}
+
 const webDiplomacyTheme = createTheme({
   breakpoints: {
     values: {
-      xs: 0,
-      sm: 414,
-      md: 834,
-      lg: 1200,
-      xl: 1500,
+      mobile: 0,
+      mobileLg: 414,
+      tablet: 834,
+      desktop: 1500,
```

Comment:
these may change, but for now, we have decided to use these names. therefore, when you use any styling by breakpoint, you'll have to refer to these names instead of the default mui names.

example:
```
[webDiplomacyTheme.breakpoints.up("desktop")]: {
    fontSize: 20,
  },
```
changes font-size in desktop breakpoint and up. mobile is first